utils.py

```diff
@@ -453,3 +454,38 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+CACHE_SLUG = None
+def get_cache_slug():
+    """Returns appropriate cache slug depending whether dev or prod. It is used
+    as a prefix in urls for images, css and script files. It is also used in
+    backend_tests to verify the presence of static resources.
+    """
+    global CACHE_SLUG # pylint: disable=global-statement
+    if CACHE_SLUG is None:
+        # pylint: disable=redefined-outer-name
```

Comment:
Put this on the right of the relevant line, otherwise you're disabling checks on everything from this line onwards. It's ok if it overflows 80 chars.